src/DataMigration/DataMigration/help/Get-AzDataMigrationAssessment.md

```diff
@@ -78,6 +78,19 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Config File Format
```

Comment:
These two files are generated from `src/DataMigration/DataMigration.Autotrest/examples/`. Please update the source and generated the help files